readme.md

```diff
@@ -363,6 +363,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Microlink](https://microlink.io)** — Microlink is a cloud browser as API. It offers Lighthouse reports on demand, making it easy to build any service on top. Similar functionality is available via the underlying open-source project named browserless.
 
+* **[Peyk](https://peyk.com)** - Peyk is a website change detection & monitoring service with the power of an advanced correlation system and a variety of modules. Peyk can track websites cookies, network requests, technologies, local & session storage, lighthouse audits, and many more besides visual tracking. Peyk is offered via free and paid plans.
```

Comment:
```suggestion
* **[Peyk](https://peyk.com)** - Peyk is a website change detection & monitoring service. Peyk can detect changes in cookies, network requests, technologies, local & session storage, lighthouse audits, screenshots and so much more. Peyk is offered via free and paid plans.
```

optional change that sounded more natural to me, but up to you :)

Reply:
Thanks for the suggestion! Glad you like it @patrickhulce :)